### ISSUE TITLE
Please make a HD version with 10X better visuals and sounds 

### ISSUE THREAD

**Issue by fisforfaheem** [VadimBoev/FlappyBird#39]

**Comment by VadimBoev** [comment]
Of course, this contradicts the original idea of this repository, but in the form of a separate branch, I think this is possible. I like the idea of HD quality, it's good

**Comment by fisforfaheem** [comment]
Thanks, Looking forward for that! please make sure its 10 to 100X better in
hd graphics!
